fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/data/FreePanic.scala

```diff
@@ -0,0 +1,146 @@
+package izumi.functional.bio.data
+
+import izumi.functional.bio.{BIOExit, BIOPanic}
+
+import scala.util.Try
+
+sealed trait FreePanic[S[_, _], +E, +A] {
+  @inline final def flatMap[B, E1 >: E](fun: A => FreePanic[S, E1, B]): FreePanic[S, E1, B] = FreePanic.FlatMapped[S, E, E1, A, B](this, fun)
+  @inline final def map[B](fun: A => B): FreePanic[S, E, B] = flatMap(a => FreePanic.pure[S, B](fun(a)))
+  @inline final def as[B](as: B): FreePanic[S, E, B] = map(_ => as)
+  @inline final def *>[B, E1 >: E](sc: FreePanic[S, E1, B]): FreePanic[S, E1, B] = flatMap(_ => sc)
+  @inline final def <*[B, E1 >: E](sc: FreePanic[S, E1, B]): FreePanic[S, E1, A] = flatMap(sc.as)
+
+  @inline final def sandbox: FreePanic[S, BIOExit.Failure[E], A] =
```

Comment:
Let's keep the same names everywhere.
```suggestion
  @inline final def sandboxExit: FreePanic[S, BIOExit.Failure[E], A] =
```